backend/FwLite/FwLiteProjectSync.Tests/SyncTests.cs

```diff
@@ -84,7 +86,18 @@ public async Task FirstSyncJustDoesAnImport()
         var fwdataEntries = await fwdataApi.GetEntries().ToArrayAsync();
         crdtEntries.Should().BeEquivalentTo(fwdataEntries,
             options => options.For(e => e.Components).Exclude(c => c.Id)
-                              .For(e => e.ComplexForms).Exclude(c => c.Id));
+                .For(e => e.ComplexForms).Exclude(c => c.Id));
+    }
+
+    [Fact]
+    public async Task SecondSyncDoesNothing()
+    {
+        var crdtApi = _fixture.CrdtApi;
+        var fwdataApi = _fixture.FwDataApi;
+        await _syncService.Sync(crdtApi, fwdataApi);
```

Comment:
that's a good point, though I think in that case I'd rather have a test specifically for dry runs, eg make sure no data is changed and like you said the change count is the same.

Reply:
alright, I changed this test so it doesn't use dry run, then I wrote some tests for dry runs specifically. While doing that I realized that our Sena3 sync tests are really only testing our import code. So I attempted to write a test to bypass the import, but now I'm running into errors because we don't import ComplexFormTypes or Writing Systems